site/zenodo_rdm/legacy/deserializers/metadata.py

```diff
@@ -24,46 +24,13 @@
 from nameparser import HumanName
 from werkzeug.local import LocalProxy
 
+from ..vocabularies.funders import FUNDER_DOI_TO_ROR
 from ..vocabularies.licenses import LEGACY_LICENSES, legacy_to_rdm
 
 record_identifiers_schemes = LocalProxy(
     lambda: current_app.config["RDM_RECORDS_IDENTIFIERS_SCHEMES"]
 )
 
-# based on
-# DOI to id https://digital-repositories.web.cern.ch/zenodo/support/operations/#openaire-grants-import
-# id to ROR: https://github.com/inveniosoftware/invenio-vocabularies/blob/master/invenio_vocabularies/config.py#L64
-FUNDER_DOI_TO_ROR = {
```

Comment:
two reasons:
1. Makes sense semantically to be in vocabularies as `LICENSES`
2. To break a dependency on invenio modules when importing it for the transform step